deliverables/README.md

```diff
@@ -12,16 +14,28 @@ Sign Up for Cloud9 and create a basic Rails project.
 ####Required
 
 What is an "interpreted" language?<br>
+A language that can be implemented/executed immediately without the use of a compiler<br>
 What is IRB?<br>
+Interactive Ruby Shell<br>
 What are Objects?<br>
+An abstract data type with abilities like inheritance<br>
 What are Methods?<br>
+Functions... Don't know why they're not named functions<br>
 What are Classes?<br>
+A template for creating objects<br>
```

Comment:
Close.  In Ruby, Classes are actually objects too.  It's a template for creating instances